pasta_eln/gui.py

```diff
@@ -254,3 +239,22 @@ def main() -> None:
 # called by python3 -m pasta_eln.gui
 if __name__ == '__main__':
   main()
```

Comment:
I see that this branch throws errors when I tried to launch the GUI, Would it not be better to have a working version committed in this branch? So that I can also do a quick test in my setup too, just a tip to avoid any breaking solution to be merged into the main branch!